### DISCUSSION TITLE
Subir pagina

### DISCUSSION THREAD

**Discussion by oscardanielM** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#310] [General]
Hola para quien quiera subir su pagina de una manera muy fácil y sencilla podrían usar 
https://www.000webhost.com/

Se registran  y ya podrían subir su pagina, desde el administrador de archivos

Espero les sirva.

**Comment by AngelCruzO** [comment]
Es bueno el host, el problema es que solo te deja hacer un sitio. Es mejor Github Pages

**Reply by aleportillo** [reply]
Puedes hacer varios (bueno un solo sitio y a partir de ahí generar varias paginas), solo subes los archivos e indicas en el url el html que quieres cargar o ver. Por ejemplo:  /pokedex.html  o  /pasteleria.html

**Reply by AngelCruzO** [reply]
Bueno si, pero no me gusta tenerlo asi. Ahorita funciona porque son paginas pequeñas, pero si eso lo quieres escalar es muy pesado puesto que necesitas mas espacio y tener un control muy bueno sobre los url y no confundirte. Por ejemplo: como haces el deploy de Vue, sin que los demas dejen de funcionar bien

**Comment by JuanSalvador73** [comment]
Ya lo conocía, pero gracias por el dato. :D
